stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -70,19 +72,32 @@ public static void main(String[] args) {
 
   @Override
   public String getBenchDescription() {
-    return "This stress bench is designed for testing the reading throughput of Fuse-based "
-        + "POSIX API. To write data, run `bin/alluxio runClass alluxio.stress.cli.fuse."
-        + "fuseIOBench --operation Write`. To read data, run `bin/alluxio runClass "
-        + "alluxio.stress.cli.fuse.fuseIOBench --operation Read`. You can further adjust "
-        + "the parameters specified below. Note that the \"--operation\" is required, the "
-        + "\"--local-path\" can be a local filesystem path or a mounted fuse path, and test files "
-        + "need to be written first before reading.\nFor example, run `bin/alluxio runClass "
-        + "alluxio.stress.cli.fuse.fuseIOBench --operation Write --local-path /mnt/alluxio-fuse"
-        + "/FuseIOTest --num-files 100 --file-size 100m --threads 32` to write the test files "
-        + "first, then run `bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench "
-        + "--operation Read --local-path /mnt/alluxio-fuse/FuseIOTest --num-files 100 "
-        + "--file-size 100m --threads 16 --warmup 15s --duration 30s` to test the reading "
-        + "throughput.\n";
+    return String.join("\n", ImmutableList.of(
+        "A stress bench for testing the reading throughput of Fuse-based POSIX API.",
+        "To run the test, write the data first, then list the files to cache the metadata, "
+            + "finally perform the read operation to test the throughput.",
+        "Note that \"--operation\" is required, the \"--local-path\" can be a local "
+            + "filesystem path or a mounted Fuse path, the test files need to be written "
+            + "first before reading, and set alluxio.user.metadata.cache.enabled=true before "
+            + "mounting the Alluxio Fuse to ensure the metadata could be cached.",
```

Comment:
This is not a requirement, but a thing we can do.
One may want to know the overall read performance, or the read performance without the metadata RPCs depending on what we are testing.